pipenv/utils.py

```diff
@@ -277,6 +277,7 @@ class PipCommand(basecommand.Command):
     pypi = PyPIRepository(
         pip_options=pip_options, use_json=True, session=session
     )
+    print(pip_options)
```

Comment:
stray debug?

Reply:
Yep, missed in my pre-commit diff grep. Removed.